exercises/hello_rubyric/config.yaml

```diff
@@ -0,0 +1,13 @@
+_external: true
+title: Hello Rubyric
+description: A simple grading example with Rubyric and Python
```

Comment:
description: this example has nothing to do with Python, right? It is only Rubyric.

Reply:
That was an old title I used. Now, I change it.